driver-core/src/test/java/com/datastax/driver/core/querybuilder/QueryBuilderTest.java

```diff
@@ -300,6 +300,24 @@ public void updateTest() throws Exception {
     }
 
     @Test(groups = "unit")
+    public void should_handle_update_if_exists() {
```

Comment:
Do you think this possibility should be removed ? I found nothing specifying it's invalid in the documentation so it could well be supported in the future. WDYT ?